Dockerfile

```diff
@@ -62,11 +62,21 @@ COPY --from=postgresql_container_source /postgresql-container/10/root/usr/libexe
 # This image must forever use UID 26 for postgres user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN yum -y --setopt=tsflags=nodocs install \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
-    yum -y module enable postgresql:10 && \
-    INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper postgresql-server postgresql-contrib" && \
+RUN if [ "$(uname -m)" != "s390x" ]; then \
+      yum -y --setopt=tsflags=nodocs install \
+         http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
+         http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
+      yum -y module enable postgresql:10 && \
+      yum -y --setopt=tsflags=nodocs install postgresql-server postgresql-contrib && \
+      rpm -V postgresql-server postgresql-contrib; \
```

Comment:
Do we need this?

Reply:
Yes. we will need this very much for non-s390x i.e x86 / ppc64le architectures. This is old code retained within !s390x conditional